### PR TITLE
Basic Menu

### PR DESCRIPTION
**Pull request by narisaraj**
Basic Menu with Setting and Help

### CODE REVIEW

File: lib/page/setting.dart [hunk 1]
```diff
@@ -0,0 +1,248 @@
...
+import 'package:flutter/material.dart';
+import 'package:meta/meta.dart';
+import 'package:flutter/rendering.dart';
+import 'package:homework_flutter/page/homepage.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:go_router/go_router.dart';
+import 'package:homework_flutter/main.dart';
+
+/// The screen of the second page.
+class Page2Screen extends StatefulWidget {
```
Comment (jimmixpert) [inline lib/page/setting.dart]: Should name class same as file name and if it is a page should ended with 'page' e.g. SettingPage

File: lib/page/setting.dart [hunk 2]
```diff
@@ -0,0 +1,248 @@
...
+            ),
+            Text('   Change Languege',
+                style: TextStyle(
+                  color: Color.fromARGB(255, 17, 251, 255),
+                  fontSize: 15,
+                  fontFamily: 'Athihi',
+                )),
+            ListTile(
+              title: Text(
+                '     English',
```
Comment (jimmixpert) [inline lib/page/setting.dart]: all related to space should use padding or sizedbox instead

File: lib/page/help.dart [hunk 3]
```diff
@@ -0,0 +1,231 @@
...
+  @override
+  Widget build(BuildContext context) => Scaffold(
+        appBar: AppBar(title: const Text(App.title)),
+        body: ListView(
+          children: [
+            SizedBox(
+              height: 20,
+            ),
+            ListTile(
+              title: Text('     FAQ',
```
Comment (jimmixpert) [inline lib/page/help.dart]: should use padding or sizedbox instead of spaces ('.      ')

**Review by jimmixpert — CHANGES_REQUESTED** [review]

File: lib/main.dart [hunk 4]
```diff
@@ -7,109 +13,64 @@ void main() {
...
 
-class _MyHomePageState extends State<MyHomePage> {
-  int _counter = 0;
+  final GoRouter _router = GoRouter(
+    errorBuilder: (context, state) => ErrorScreen(error: state.error),
+    routes: <GoRoute>[
+      GoRoute(
+        routes: <GoRoute>[
+          GoRoute(
+            path: 'page2',
```
Comment (jimmixpert) [inline lib/main.dart]: should name route in an appropriated way e.g. home, settings, help

**Review by jimmixpert — CHANGES_REQUESTED** [review]

File: pubspec.yaml [hunk 5]
```diff
@@ -89,3 +93,19 @@ flutter:
...
+    - asset: fonts/Australia Custom.ttf
+  - family: Pacifico
+    fonts:
+    - asset: fonts/Pacifico.ttf
+  - family: ostrich
+    fonts:
+    - asset: fonts/ostrich-regular.ttf
+  - family: Antonio
+    fonts:
+    - asset: fonts/Antonio-Bold.ttf
```
Comment (krisadamqdc) [inline pubspec.yaml]: should have new line

File: lib/page/homepage.dart [hunk 6]
```diff
@@ -0,0 +1,52 @@
+import 'package:flutter/material.dart';
+import 'package:go_router/go_router.dart';
+import 'package:homework_flutter/main.dart';
+
+class HomePage extends StatelessWidget {
+  const HomePage({Key? key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) => Scaffold(
```
Comment (krisadamqdc) [inline lib/page/homepage.dart]: should not use Lamda should use {} directly

File: pubspec.yaml (hunk 5 above)
Comment (narisaraj) [reply pubspec.yaml]: edit complete

File: lib/page/homepage.dart (hunk 6 above)
Comment (narisaraj) [reply lib/page/homepage.dart]: edit complete

File: pubspec.yaml [hunk 7]
```diff
@@ -89,3 +110,4 @@ flutter:
   #
   # For details regarding fonts from package dependencies,
   # see https://flutter.dev/custom-fonts/#from-packages
+
```
Comment (krisadamqdc) [inline pubspec.yaml:113]: looks like you have spaces so it wont detect as empty new line

**Review by krisadamqdc — APPROVED** [review]

**Pull request merged**